ChangeLog

```diff
@@ -1,3 +1,17 @@
+* Wed Aug 24 2022 Christopher M Cantalupo <christopher.m.cantalupo@intel.com> v2.0.0
+- Official v2.0.0 release tag.
+- Provides the GEOPM Systemd Service.
+- Support for GPUs from Intel and NVIDIA.
+- Higher performance / lower latency profile interface.
+- More consistent naming scheme for PlatformIO signals and controls.
+- Extended set of signals and controls provided by PlatformIO.
+- Removed msr-safe requirement though GEOPM Service features.
+- Support for new HPC runtime launchers.
+- Flexible YAML report generation and parsing that may contain arbitrary content.
+- Extended python interface support including Reporter features.
+- Python based agents for prototyping runtime algorithms that do not require application feedback.
+- Documentation and web page improvements.
+- Other improvements and feature additions.
```

Comment:
Line about EE agent removal
Line about new IOGroups: SST, CNL, L0, NVML(?), DCGM(?)
Python 2 support is fully deprecated - only 3 is supported going forward.
CPU support: SKX, ICX, CLX(?)